src/sinks/prometheus/remote_write.rs

```diff
@@ -27,6 +27,15 @@ use std::task;
 use tower::ServiceBuilder;
 use vector_core::ByteSizeOf;
 
+#[derive(Clone, Copy, Debug, Default)]
+pub struct PrometheusRemoteWriteDefaultBatchSettings;
+
+impl SinkBatchSettings for PrometheusRemoteWriteDefaultBatchSettings {
+    const MAX_EVENTS: Option<usize> = Some(1_000);
+    const MAX_BYTES: Option<usize> = None;
+    const TIMEOUT_SECS: NonZeroU64 = unsafe { NonZeroU64::new_unchecked(1) };
+}
```

Comment:
Same here, the implementation changed substantially from the previously configured values.

Reply:
Ugh, my bad. 👍🏻